Resources/Prototypes/Stacks/Materials/materials.yml

```diff
@@ -27,3 +27,9 @@
   name: cotton
   icon: /Textures/Objects/Materials/materials.rsi/cotton.png
   spawn: MaterialCotton1
+
+- type: stack
+  id: FissionFuel
+  name: fission fuel
+  icon: /Textures/Objects/Materials/materials.rsi/diamond.png
```

Comment:
would prefer this to use a new texture

Reply:
Just remove fission fuel, it's out-of-scope

Reply:
yeah I will do that